### ISSUE TITLE
compile error with JDK15

### ISSUE THREAD

**Issue by xuwei-k** [playframework/playframework#10478]
### JDK

```
$ java --version
openjdk 15 2020-09-15
OpenJDK Runtime Environment (build 15+36-1562)
OpenJDK 64-Bit Server VM (build 15+36-1562, mixed mode, sharing)
```

### Expected Behavior

compile success

### Actual Behavior

```
[error] /Users/kenji/temp/playframework/transport/server/play-server/src/main/scala/play/core/server/ssl/CertificateGenerator.scala:43:19: value sha256WithRSAEncryption_oid is not a member of object sun.security.x509.AlgorithmId
[error]       AlgorithmId.sha256WithRSAEncryption_oid
[error]                   ^
[error] /Users/kenji/temp/playframework/transport/server/play-server/src/main/scala/play/core/server/ssl/CertificateGenerator.scala:64:19: value sha256WithRSAEncryption_oid is not a member of object sun.security.x509.AlgorithmId
[error]       AlgorithmId.sha256WithRSAEncryption_oid
[error]                   ^
[error] /Users/kenji/temp/playframework/transport/server/play-server/src/main/scala/play/core/server/ssl/CertificateGenerator.scala:90:93: value md5WithRSAEncryption_oid is not a member of object sun.security.x509.AlgorithmId
[error]     generateCertificate(dn, pair, Date.from(from), Date.from(to), "MD5WithRSA", AlgorithmId.md5WithRSAEncryption_oid)
[error]                                                                                             ^
[error] three errors found
```

- https://github.com/playframework/playframework/blob/810fca22fb1a5b8111fcf84d524349f9a66c22ab/transport/server/play-server/src/main/scala/play/core/server/ssl/CertificateGenerator.scala
- https://github.com/openjdk/jdk/commit/080b3b83ebffe5149fbc9ac48e921fb5

`sun.security.x509.AlgorithmId.sha256WithRSAEncryption_oid` and `sun.security.x509.AlgorithmId.md5WithRSAEncryption_oid` removed(?) since JDK15

```
Welcome to Scala 2.13.3 (OpenJDK 64-Bit Server VM, Java 14).
Type in expressions for evaluation. Or try :help.

scala> sun.security.x509.AlgorithmId.md5WithRSAEncryption_oid
val res0: sun.security.util.ObjectIdentifier = 1.2.840.113549.1.1.4
```

```
Welcome to Scala 2.13.3 (OpenJDK 64-Bit Server VM, Java 15).
Type in expressions for evaluation. Or try :help.

scala> sun.security.x509.AlgorithmId.md5WithRSAEncryption_oid
                                     ^
       error: value md5WithRSAEncryption_oid is not a member of object sun.security.x509.AlgorithmId
```

### Reproducible Test Case

- checkout playframework/playframework
- `sbt compile` with JDK15

**Comment by raboof** [comment]
I suspect even if we fix this compile error, it would still not work since ssl-config also uses the now-removed API: https://github.com/lightbend/ssl-config/pull/269

**Comment by SethTisue** [comment]
is this resolved now that https://github.com/playframework/playframework/pull/10487 is merged? (I might be confused.)

**Comment by ignasi35** [comment]
> is this resolved now that #10487 is merged? (I might be confused.)

No, it the same error in different places. `CertificateGenerator` still uses deprecated API. should be same fix than https://github.com/lightbend/ssl-config/pull/269, though.

**Comment by mkurz** [comment]
PR with fix: #10729
